### PR TITLE
Updating to bring Cocoapods dependency up to date

### LINKED ISSUE #1070: #import "ATLAddressBarViewController.h" not found
My app support iOS 7
Using Atlas (1.0.11)
Using LayerKit (0.16.0)

Find Attached ..

untitled

i got this error

### PR DESCRIPTION
**Pull request by mathmatrix828**
Resolves #1070

### CODE REVIEW

**Comment by mathmatrix828** [comment]
Note that checks have failed due to TravisCI issue, not code

**Comment by maju6406** [comment]
Thank @mathmatrix828 . I don't think the changes to Atlas.xcodeproj/project.pbxproj are necessary to get Cocoapods working. I believe they're just project changes that happen after you run Pod install. Also in the interested of keeping this PR clean can you put the changes to "Examples/Storyboard/StoryboardAppDelegate.m" in a separate PR? Thanks!

**Comment by mathmatrix828** [comment]
Please note that most of the changes are due to re-linking the Cocoapods dependencies. And I can go ahead and roll back those changes as I don't think they're necessary

**Comment by maju6406** [comment]
Yes @mathmatrix828 please roll back those changes to the project files.

File: Podfile [hunk 1]
```diff
@@ -5,27 +5,42 @@ source 'https://github.com/CocoaPods/Specs.git'
...
 
-link_with 'Programmatic', 'Storyboard'
+# NOTE: link_with -> abstract_target (inheritance) ref https://guides.cocoapods.org/syntax/podfile.html#abstract_target
+abstract_target 'Atlas-iOS' do
+  pod 'Atlas', path: '.'
+  pod 'OCMock'
+  pod 'LayerKit'
 
-pod 'Atlas', path: '.'
+  target 'Atlas'
```
Comment (kcoleman731) [inline Podfile]: No need to introduce an `Atlas` target here.

File: Podfile [hunk 2]
```diff
@@ -5,27 +5,42 @@ source 'https://github.com/CocoaPods/Specs.git'
 
 use_frameworks!
 
-link_with 'Programmatic', 'Storyboard'
+# NOTE: link_with -> abstract_target (inheritance) ref https://guides.cocoapods.org/syntax/podfile.html#abstract_target
+abstract_target 'Atlas-iOS' do
+  pod 'Atlas', path: '.'
+  pod 'OCMock'
```
Comment (kcoleman731) [inline Podfile]: OCMock is not a dep of the `Programmatic` or `Storyboard` targets...It should only be included for the test targets.

File: Podfile [hunk 3]
```diff
@@ -5,27 +5,42 @@ source 'https://github.com/CocoaPods/Specs.git'
...
+  # Storyboard application
+  target 'Storyboard'
+  target 'StoryboardTests' do
+    pod 'Expecta'
+    pod 'KIFViewControllerActions', git: 'https://github.com/blakewatters/KIFViewControllerActions.git'
+    pod 'LYRCountDownLatch', git: 'https://github.com/layerhq/LYRCountDownLatch.git'
+    pod 'KIF'
+  end
+
+  target 'UnitTests'
```
Comment (kcoleman731) [inline Podfile]: Unit tests requires expects.

**Comment by kcoleman731** [comment]
@mathmatrix828 Thanks for the PR! I just left a few comments on the actual Podfile. If you can make those changes we will get this merged!

**Comment by kcoleman731** [comment]
@mathmatrix828 we had to merge a few PRs on Friday to bump Atlas to `CocoaPods 1.0.0` compatibility for our release. Thank you very much for your PR but I am going to close as it is no longer necessary.

**Pull request closed without merging**